samples/BikeSharingApp/README.md

```diff
@@ -1,4 +1,79 @@
-# Bike Sharing Sample: Iteratively Develop and Debug Microservices in Kubernetes
+# Deploy the Bike Sharing sample application to Azure Kubernetes Service
 
-## Quickstart
-[Quickstart: Team development on Kubernetes using Azure Dev Spaces](https://docs.microsoft.com/en-us/azure/dev-spaces/quickstart-team-development).
\ No newline at end of file
+*Bike Sharing* is a microservices-based sample application that helps showcase the sandboxing capabilities of [Azure Dev Spaces](https://aka.ms/devspaces). 
+
+Follow the steps below to deploy this sample app to Azure Kubernete Service (AKS).
+
+## Prerequisites
+* [Azure subscription](https://azure.microsoft.com/free)
+* [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [Helm 2.13 (or greater)](https://github.com/helm/helm/blob/master/docs/install.md)
```

Comment:
>https://github.com/helm/helm/blob/master/docs/install.md [](start = 27, length = 56)

I'm pretty sure our official documentation has the same problem, but this link doesn't seem to be valid anymore. :(
It returns me a 404 error page from GitHub.
It seems that the new link is: [https://github.com/helm/helm#install](https://github.com/helm/helm#install)

Reply:
Good catch! I updated the link.